# Relative Pose Filter Parameters
# Rates
update_freq: 100.0 # Hz
measurement_freq: 15.0 # Hz
measurement_delay: 0.030 # s
measurement_delay_max: 0.200 # s
dyn_measurement_delay_offset: 0.005 # s

# Topics
IMU_topic: "/drone/imu"

# Noise Values
Q_a_diag: [0.0005,0.0005,0.0005] # Dropped by an order of magnitude
Q_w_diag: [0.00005,0.00005,0.00005]
Q_ab_diag: [5.0E-5,5.0E-5,5.0E-5]
Q_wb_diag: [5.0E-6,5.0E-6,5.0E-6]

R_r_diag: [0.015,0.015,0.020] # 0.005,0.005,0.015
R_ang_diag: [0.0015,0.0015,0.04] # 0.0025,0.0025,0.025

accel_bias_static: [0.0,0.0,0.0]
gyro_bias_static: [0.0,0.0,0.0]

# Camera Parameters
r_v_cv: [0,0,-0.073]
q_vc: [0.70711,-0.70711,0,0]

camera_K: [241.4268,0,376.5,0,241.4268,240.5,0,0,1]
camera_width: 752
camera_height: 480

# Mahony Filter
kp: 0.1
ki: 0.05
mahony_accel_bias_init: [-0.205,-0.17,0.065]
mahony_gyro_bias_init: [-0.002,0.006,0.0]

# Target Configuration
n_tags: 1
tag_in_view_margin: 0.02

tag_widths: [0.8] # m
tag_positions: [0,0,0]

# Flags/Options
limit_measurement_freq: True
corner_margin_enbl: True
est_bias: True
direct_orien_method: True
multirate_ekf: True
dynamic_meas_delay: True
